### PR TITLE
Feature/deprecate/set wildfire

### PR DESCRIPTION
**Pull request by samluethi**
update deprecated set methods for the wildfire hazard as well as the corresponding impact function

### CODE REVIEW

File: climada_petals/entity/impact_funcs/test/test_wf.py [hunk 1]
```diff
@@ -47,6 +47,24 @@ def test_default_values_FIRMS_pass(self):
         paa = i_n**3 / (1 + i_n**3)
         self.assertTrue(np.array_equal(imp_fun.paa, paa))
         self.assertTrue(np.array_equal(imp_fun.mdd, np.ones(len(paa))))
+        
+    def test_from_default_values_FIRMS_pass(self):
+        """Compute mdr interpolating values. For the calibrated function"""
+        imp_fun = ImpfWildfire()
+        imp_fun = imp_fun.from_default_FIRMS(i_half_check)
```
Comment (Evelyn-M) [inline climada_petals/entity/impact_funcs/test/test_wf.py:54]:
```suggestion
        imp_fun = ImpfWildfire.from_default_FIRMS(i_half_check)
```

File: climada_petals/hazard/test/test_wildfire.py [hunk 2]
```diff
@@ -52,7 +52,8 @@ def def_ori_centroids(firms, centr_res_factor):
         firms['longitude'].min() : firms['longitude'].max() : complex(0, nb_centr_lon)]). \
         reshape(2, nb_centr_lat*nb_centr_lon).transpose()
 
-    centroids = Centroids.from_lat_lon(coord[:, 0], coord[:, 1])
+    centroids = Centroids()
+    centroids.set_lat_lon(coord[:, 0], coord[:, 1])
```
Comment (Evelyn-M) [inline climada_petals/hazard/test/test_wildfire.py:56]:
```suggestion
    centroids = Centroids.from_lat_lon(coord[:, 0], coord[:, 1])
```
Why did you change this back to the old set method? The new way is as was.

File: climada_petals/hazard/wildfire.py [hunk 3]
```diff
@@ -635,7 +662,8 @@ def _firms_centroids_creation(df_firms, res_data, centr_res_factor):
         -------
         centroids : Centroids
         """
-        centroids = Centroids.from_pnt_bounds((df_firms['longitude'].min(), \
+        centroids = Centroids()
+        centroids.set_raster_from_pnt_bounds((df_firms['longitude'].min(), \
```
Comment (Evelyn-M) [inline climada_petals/hazard/wildfire.py:666]: Also here, this is deprecated.
```suggestion
        centroids = Centroids.from_pnt_bounds((df_firms['longitude'].min(), \
```

File: climada_petals/test/test_wildfire.py [hunk 4]
```diff
@@ -84,11 +112,35 @@ def test_hist_fire_season_firms_pass(self):
...
+        self.assertAlmostEqual(wf.intensity[0, 4005], 318.5)
+        self.assertAlmostEqual(wf.intensity[0, 8000], 0.0)
+        self.assertAlmostEqual(wf.fraction.max(), 1.0)
+        self.assertAlmostEqual(wf.fraction.min(), 0.0)
 
     def test_proba_fire_season_pass(self):
         """ Test probabilistic set_probabilistic_event_year_set """
         wf = WildFire()
-        wf.set_hist_fire_seasons_FIRMS(TEST_FIRMS)
+        wf = wf.from_hist_fire_seasons_FIRMS(TEST_FIRMS)
```
Comment (Evelyn-M) [inline climada_petals/test/test_wildfire.py:143]: Delete prior line and initalize like so:

```suggestion
        wf = WildFire.from_hist_fire_seasons_FIRMS(TEST_FIRMS)
```

File: climada_petals/test/test_wildfire.py [hunk 5]
```diff
@@ -110,7 +162,7 @@ def test_proba_fire_season_pass(self):
     def test_summarize_fires_to_seasons_pass(self):
         """ Test probabilistic set_probabilistic_event_year_set """
         wf = WildFire()
-        wf.set_hist_fire_FIRMS(TEST_FIRMS)
+        wf = wf.from_hist_fire_FIRMS(TEST_FIRMS)
```
Comment (Evelyn-M) [inline climada_petals/test/test_wildfire.py:165]: Delete prior line and initialize like so:
```suggestion
        wf = WildFire.from_hist_fire_FIRMS(TEST_FIRMS)
```

**Review by Evelyn-M — CHANGES_REQUESTED** [review]

File: climada_petals/hazard/test/test_wildfire.py (hunk 2 above)
Comment (samluethi) [reply climada_petals/hazard/test/test_wildfire.py:56]: whoop sorry, I think I messed something up with the climada on my laptop =)
